### PR TITLE
Fix GH issues in commit message, disallow bugzilla

### LINKED ISSUE #123: Commit message checking is broken for GH issues

### PR DESCRIPTION
**Pull request by avishayt**
fixes #123

### CODE REVIEW

**Review by mangelajo — APPROVED** [review]

File: hack/check-commit-message.sh [hunk 1]
```diff
@@ -4,11 +4,12 @@ set -o nounset
 
 commit_file=${1}
 commit_message="$(cat ${commit_file})"
-valid_commit_regex='^(NO-ISSUE|Merge|Revert|((Bug |[A-Z]+-|#)[0-9]+:))'
+valid_commit_regex='^(NO-ISSUE|Merge|Revert|([A-Z]+-[0-9]+:))'
+valid_github_regex='^((Fixes|fixes|Closes|closes): #[0-9]+)'
```
Comment (mangelajo) [inline hack/check-commit-message.sh:8]: Please note here you require a ' ' after :

**Review by mangelajo — CHANGES_REQUESTED** [review]
see comments

File: hack/check-commit-message.sh [hunk 2]
```diff
@@ -4,11 +4,12 @@ set -o nounset
...
 
 error_msg="""Aborting commit: ${commit_message}
 ---
-Your commit message should start with a JIRA issue ('JIRA-1111'), a GitHub issue ('#39'), or a BugZilla issue ('Bug 123')
+Your commit message should start with a JIRA issue ('JIRA-1111') or a GitHub issue ('Fixes:#39')
```
Comment (mangelajo) [inline hack/check-commit-message.sh:12]:
```suggestion
Your commit message should start with a JIRA issue ('JIRA-1111') or a GitHub issue ('Fixes #39')
```
 I propose this format instead.

File: hack/check-commit-message.sh (hunk 1 above)
Comment (mangelajo) [reply hack/check-commit-message.sh:8]: I would also accept Issue #xx

because it can be related to an issue, but neither fixes it or closes it.

Comment (avishayt) [reply hack/check-commit-message.sh]: Done

File: hack/check-commit-message.sh (hunk 2 above)
Comment (avishayt) [reply hack/check-commit-message.sh]: Done

**Review by mangelajo — APPROVED** [review]

**Pull request merged**